src/imars3d/backend/__main__.py

```diff
@@ -0,0 +1,52 @@
+"""
```

Comment:
Do you want to install this as an actual entrypoint?

By adding this to the `setup.cfg`

```python
[options.entry_points]
console_scripts =
    imars3d = imars3d.backend.__main__:main 
```

that way you can run `imars3d tests/data/json/good_non_interactive.json` instead of `python -m imars3d.backend tests/data/json/good_non_interactive.json`

Reply:
It doesn't show up when I do `conda develop .` and `python -m build --wheel --no-isolation` doesn't put in the installer. The docs agree with you though.

Reply:
Installing with `pip` works